src/app/shared/dialogs/dialogs-form.component.html

```diff
@@ -4,7 +4,10 @@ <h1 mat-dialog-title><span><mat-icon>create</mat-icon>{{title}}</span></h1>
     <div *ngFor="let field of fields">
       <!-- input field -->
       <mat-form-field *ngIf="field.type === 'textbox'" class="full-width">
-        <input matInput placeholder="{{field.placeholder}}" type="{{field.inputType || 'text'}}" formControlName="{{field.name}}" required="{{field.required}}">
+        <input matInput placeholder="{{field.placeholder}}" type="{{(field.togglePassword && field.inputType == 'password') ? 'password' : 'text'}}" formControlName="{{field.name}}" required="{{field.required}}">
+          <button *ngIf="field.inputType" type="button" mat-icon-button matSuffix (click)="field.togglePassword = !field.togglePassword">
```

Comment:
This shows the toggle password icon button for any `<input>` field in the form.  It should only be for passwords.